### DISCUSSION TITLE
Access to create budget alerts in compartment level

### DISCUSSION THREAD

**Discussion by vj4a** [Sunbird-Ed/Community#654] [General Discussion]
Hello @santhosh-tg @gandham-santhosh,

We were unable to create budget alerts even under compartment SBEd-Sandbox level. Kindly provide for the same.

![image](https://github.com/Sunbird-Ed/Community/assets/63919015/cf9da7bc-f184-4442-848d-e72815097b51)

**Comment by mohitga** [comment]
@santhosh-tg can you help @vj4a in the above query.

**Reply by santhosh-tg** [reply]
@mohitga @vj4a Replied via email thread

**Comment by mohitga** [comment]
@vj4a  can you mark this issue answered?

**Reply by vj4a** [reply]
@mohitga , This is resolved.
